numpy/core/_multiarray_umath.py

```diff
@@ -56,6 +57,11 @@ def __getattr__(attr_name):
                 it indicates the need for a new pybind11 release.
 
                 """)
+        msg += "Traceback (most recent call last):"
+        for line in traceback.format_stack():
```

Comment:
```suggestion
        for line in traceback.format_stack()[:-1]:
```
Maybe skip us, so that the first line is the one that fails to import NumPy?